File: cpp/Disk.cpp
#include <iostream>

using namespace std;

class Disk
{
private:
    string type;
    int capacity;
    int price;

public:
    Disk() {}
    Disk(string type, int capacity, int price)
    {
        this->type = type;
        this->capacity = capacity;
        this->price = price;
    }

    string getType()
    {
        return this->type;
    }
    void setType(string type)
    {
        this->type = type;
    }

    int getCapacity()
    {
        return this->capacity;
    }
    void setCapacity(int capacity)
    {
        this->capacity = capacity;
    }

    int getPrice()
    {
        return this->price;
    }
    void setPrice(int price)
    {
        this->price = price;
    }

    void printDisk()
    {
        cout << "Disk Type: " << this->type << endl;
        cout << "Disk Capacity: " << this->capacity << " GB\n";
        cout << "Disk Price: " << this->price << endl;
    }
    ~Disk() {}
};

File: python/Disk.py
class Disk:
    def __init__(self, type = "", capacity = 0, price = 0):
        self.type = type
        self.capacity = capacity
        self.price = price

    def getType(self):
        return self.type

    def setType(self, type):
        self.type = type

    def getCapacity(self):
        return self.capacity

    def setCapacity(self, capacity):
        self.capacity = capacity
    
    def getPrice(self):
        return self.price

    def setPrice(self, price):
        self.price = price

    def printDisk(self):
        print("Disk Type: {}".format(self.type))
        print("Disk Capacity: {} GB".format(self.capacity))
        print("Disk Price: {}".format(self.price))